esp/src/eclwatch/PreflightDetailsWidget.js

```diff
@@ -4,22 +4,25 @@ define([
     "dojo/i18n",
     "dojo/i18n!./nls/hpcc",
     "dojo/_base/array",
+    "dojo/dom-class",
+    "dojo/dom-construct",
+    "dojo/dom-geometry",
+    "dojo/_base/window",
+
+    "dojox/lang/functional",
```

Comment:
I would rather not pull this is in if it can be avoided (the same is true for all dojox functionality)